### PR TITLE
Introduce linter incl. workflow

### PR DESCRIPTION
**Pull request by KristjanESPERANTO**
Here is my suggestion for linting :slightly_smiling_face: As a template I have taken the linting of the module [MMM-PublicTransportHafas](https://github.com/KristjanESPERANTO/MMM-PublicTransportHafas).

I have not yet dealt with the linting messages. I think that is a step that should happen after merging this PR.

What do you think?

### CODE REVIEW

**Comment by deg0nz** [comment]
Hey, thank you very much! This is very much appreciated :)

It looks good to me. I will merge it.

File: .github/workflows/automated-tests.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    branches: [master, develop]
+  pull_request:
+    branches: [master, develop]
+jobs:
+  run-lint:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    strategy:
+      matrix:
+        node-version: ["16", "17"]
```
Comment (rejas) [inline .github/workflows/automated-tests.yaml:13]: sorry, a little late to the PR party ;-) why run against two versions of node? normally that makes sense when running real tests but here its jsut linting and that wont change much between node versions. just makes it run longer and wastes cpu time IMO

**Review by rejas — COMMENTED** [review]
but always nice to see other people liking nicely linted code :-D

File: .github/workflows/automated-tests.yaml (hunk 1 above)
Comment (deg0nz) [reply .github/workflows/automated-tests.yaml:13]: Yeah, that's actually right. I didn't see this.

Comment (deg0nz) [reply .github/workflows/automated-tests.yaml:13]: Then I think it would make sense to run this with the LTS version (16)

Comment (rejas) [reply .github/workflows/automated-tests.yaml:13]: Or even freshly released v18?

Comment (KristjanESPERANTO) [reply .github/workflows/automated-tests.yaml:13]: True. Since it is only about linting, one version is enough. I would also like to try it with v18.

**Pull request merged**